include/jemalloc/internal/test_hooks.h

```diff
@@ -6,14 +6,16 @@ extern JEMALLOC_EXPORT void (*test_hooks_libc_hook)();
 
 #define JEMALLOC_HOOK(fn, hook) ((void)(hook != NULL && (hook(), 0)), fn)
```

Comment:
How about we define this to be simply `fn` in the non testing case, and also rename the macro to `JEMALLOC_TEST_HOOK`? Might be a bit easier to read.

Reply:
Agree, these are good suggestions. Addressed.